core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -1150,4 +1192,43 @@ class TestCaseWriter {
                 && !printableContent.contains("logged")
                 && !printableContent.contains("""\w+:\d{4,5}""".toRegex()))
     }
+
+    fun getPrintableInputGenes(printableInputGene: MutableList<String>): String {
+        var printableInputGenes = ""
+        for (elt in printableInputGene) {
+            printableInputGenes = "$elt,$printableInputGenes"
+        }
```

Comment:
this can be simplified with something like `printableInputGene.joinToString(",")`  (not sure of the syntax though)

Reply:
Done.